Cargo.toml

```diff
@@ -93,17 +93,17 @@ sea-query = "0.28.1"
 serde = "1.0.137"
 serde_json = "1.0.81"
 serial_test = "2.0.0"
-solana-account-decoder = "~1.17"
-solana-client = "~1.17"
-solana-geyser-plugin-interface = "~1.17"
-solana-program = "~1.17"
-solana-sdk = "~1.17"
-solana-transaction-status = "~1.17"
+solana-account-decoder = "1.18.11"
+solana-client = "1.18.11"
+solana-geyser-plugin-interface = "1.18.11"
+solana-program = "1.18.11"
+solana-sdk = "1.18.11"
+solana-transaction-status = "1.18.11"
```

Comment:
Not sure if RPC community is ready to update to 1.18 FYI

Reply:
Updated smart contracts migrated to this solana version, so we must use it here or find some detours to avoid conflicts in import versions. Or maybe we can talk with the community and notify that in the near future, it will be necessary to update the solana version?

Reply:
ok that seems fine